labkey/query.py

```diff
@@ -316,7 +316,7 @@ def select_rows(
     filter_array: List[QueryFilter] = None,
     container_path: str = None,
     columns=None,
-    max_rows: int = None,
+    max_rows: int = -1,
```

Comment:
Down on line 376, should it be `if max_rows is not -1`?

Reply:
Seperately, should the `@functools.wraps` part down on line 525 be updated to -1 as well?

Reply:
For 376 - No, it needs to remain that way or we'll never pass the default value.
For 525 - Yes, good catch.